diff --git a/node_modules/@material-tailwind/react/dist/chunk-BGJMALSI.js b/node_modules/@material-tailwind/react/dist/chunk-BGJMALSI.js
index ba4ee13..950765c 100644
--- a/node_modules/@material-tailwind/react/dist/chunk-BGJMALSI.js
+++ b/node_modules/@material-tailwind/react/dist/chunk-BGJMALSI.js
@@ -44,7 +44,7 @@ var selectTriggerTheme = {
       }
     )
   },
-  baseStyle: "flex items-center gap-4 justify-between h-max w-full outline-none focus:outline-none text-foreground bg-transparent ring-transparent border border-surface transition-all duration-300 ease-in disabled:opacity-50 disabled:pointer-events-none data-[error=true]:border-error data-[success=true]:border-success select-none text-start data-[shape=pill]:rounded-full [&_data-slot=placeholder]:text-foreground/60",
+  baseStyle: "flex items-center gap-4 justify-between h-max w-full outline-none focus:outline-none text-foreground bg-transparent ring-transparent border border-surface transition-all duration-300 ease-in disabled:opacity-50 disabled:pointer-events-none data-[error=true]:border-error data-[success=true]:border-success select-none text-start data-[shape=pill]:rounded-full [&_[data-slot=placeholder]]:text-foreground/60",
   size: {
     sm: "text-sm rounded-md py-1.5 px-2 ring shadow-sm",
     md: "text-sm rounded-md py-2 px-2.5 ring shadow-sm",
diff --git a/node_modules/@material-tailwind/react/dist/chunk-BGYX7VND.js b/node_modules/@material-tailwind/react/dist/chunk-BGYX7VND.js
index 30ca2dc..20cb52f 100644
--- a/node_modules/@material-tailwind/react/dist/chunk-BGYX7VND.js
+++ b/node_modules/@material-tailwind/react/dist/chunk-BGYX7VND.js
@@ -42,7 +42,7 @@ var ratingTheme = {
     ),
     readonly: false
   },
-  baseStyle: "inline-flex items-center [&_data-slot=icon]:w-5 [&_data-slot=icon]:h-5 [&_data-slot=icon]:text-inherit [&_data-slot=icon]:cursor-pointer",
+  baseStyle: "inline-flex items-center [&_[data-slot=icon]]:w-5 [&_[data-slot=icon]]:h-5 [&_[data-slot=icon]]:text-inherit [&_[data-slot=icon]]:cursor-pointer",
   color: {
     primary: "text-primary",
     secondary: "text-secondary",
diff --git a/node_modules/@material-tailwind/react/src/theme/rating.tsx b/node_modules/@material-tailwind/react/src/theme/rating.tsx
index 4f3ac2e..a355773 100644
--- a/node_modules/@material-tailwind/react/src/theme/rating.tsx
+++ b/node_modules/@material-tailwind/react/src/theme/rating.tsx
@@ -39,7 +39,7 @@ export const ratingTheme = {
     readonly: false,
   },
   baseStyle:
-    "inline-flex items-center [&_data-slot=icon]:w-5 [&_data-slot=icon]:h-5 [&_data-slot=icon]:text-inherit [&_data-slot=icon]:cursor-pointer",
+    "inline-flex items-center [&_[data-slot=icon]]:w-5 [&_[data-slot=icon]]:h-5 [&_[data-slot=icon]]:text-inherit [&_[data-slot=icon]]:cursor-pointer",
   color: {
     primary: "text-primary",
     secondary: "text-secondary",
diff --git a/node_modules/@material-tailwind/react/src/theme/select.tsx b/node_modules/@material-tailwind/react/src/theme/select.tsx
index b858002..a169ea3 100644
--- a/node_modules/@material-tailwind/react/src/theme/select.tsx
+++ b/node_modules/@material-tailwind/react/src/theme/select.tsx
@@ -43,7 +43,7 @@ export const selectTriggerTheme = {
     ),
   },
   baseStyle:
-    "flex items-center gap-4 justify-between h-max w-full outline-none focus:outline-none text-foreground bg-transparent ring-transparent border border-surface transition-all duration-300 ease-in disabled:opacity-50 disabled:pointer-events-none data-[error=true]:border-error data-[success=true]:border-success select-none text-start data-[shape=pill]:rounded-full [&_data-slot=placeholder]:text-foreground/60",
+    "flex items-center gap-4 justify-between h-max w-full outline-none focus:outline-none text-foreground bg-transparent ring-transparent border border-surface transition-all duration-300 ease-in disabled:opacity-50 disabled:pointer-events-none data-[error=true]:border-error data-[success=true]:border-success select-none text-start data-[shape=pill]:rounded-full [&_[data-slot=placeholder]]:text-foreground/60",
   size: {
     sm: "text-sm rounded-md py-1.5 px-2 ring shadow-sm",
     md: "text-sm rounded-md py-2 px-2.5 ring shadow-sm",
